copier.yml

```diff
@@ -14,8 +14,4 @@ min_python:
 max_python:
   type: str
   help: Maximum Python version that is known to work?
-github_linux_image:
-  default: ubuntu-20.04
-  type: str
-  help: What OS image should be used for the github action?
```

Comment:
Instead of this, we will simply update this in the template, I don't see a strong need for changing this in individual template instances.